pkg/resources/deployments/console-deployment.go

```diff
@@ -32,12 +32,6 @@ func consoleEnvVars(t *miniov1.Tenant) []corev1.EnvVar {
 			Value: t.MinIOServerEndpoint(),
 		},
 	}
-	if t.TLS() {
-		envVars = append(envVars, corev1.EnvVar{
-			Name:  "CONSOLE_MINIO_SERVER_TLS_SKIP_VERIFICATION",
-			Value: "on", // FIXME: should be trusted
```

Comment:
is it now trusted? @Alevsk

Reply:
that env variable doesnt exists anymore

Reply:
That env variable doesnt exists in current console version, i'll replace that code with `CONSOLE_MINIO_SERVER_TLS_ROOT_CAS` loading the minio certificate, ill do that on a different PR

Reply:
We need to make a release so not sure if that needed or not @dvaldivia @Alevsk

Reply:
I'm adding the support for `CONSOLE_MINIO_SERVER_TLS_ROOT_CAS ` on this pr Harsha, its better to do that instead of creating another pr

Reply:
fixes